src/main/java/core/basesyntax/AreaCalculable.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface AreaCalculable {
```

Comment:
in this case, this will be more clear naming
```suggestion
public interface AreaCalculator {
```